src/OpenTelemetry.AutoInstrumentation/Configurations/EnvironmentConfigurationMetricHelper.cs

```diff
@@ -53,13 +53,18 @@ private static MeterProviderBuilder SetExporter(this MeterProviderBuilder builde
             Wrappers.AddConsoleExporter(builder, pluginManager);
         }
 
-        return settings.MetricExporter switch
+        foreach (var metricExporter in settings.MetricExporters)
         {
-            MetricsExporter.Prometheus => Wrappers.AddPrometheusHttpListener(builder, pluginManager),
-            MetricsExporter.Otlp => Wrappers.AddOtlpExporter(builder, settings, pluginManager),
-            MetricsExporter.None => builder,
-            _ => throw new ArgumentOutOfRangeException($"Metrics exporter '{settings.MetricExporter}' is incorrect")
-        };
+            builder = metricExporter switch
+            {
+                MetricsExporter.Prometheus => Wrappers.AddPrometheusHttpListener(builder, pluginManager),
+                MetricsExporter.Otlp => Wrappers.AddOtlpExporter(builder, settings, pluginManager),
+                MetricsExporter.None => builder,
```

Comment:
Why do we even return `MetricsExporter.None`?

Reply:
We can probably return empty collection when `none` is used.